slang-gfx.h

```diff
@@ -1345,6 +1449,21 @@ class IResourceCommandEncoder : public ICommandEncoder
         IBufferResource* src,
         size_t srcOffset,
         size_t size) = 0;
+    virtual SLANG_NO_THROW void SLANG_MCALL copyTexture(
+        ITextureResource* dst,
+        ITextureResource::SubresourceRange dstSubresource,
+        ITextureResource::Offset3D dstOffset,
+        ITextureResource* src,
+        ITextureResource::SubresourceRange srcSubresource,
+        ITextureResource::Offset3D srcOffset,
+        ITextureResource::Size extend) = 0;
```

Comment:
I think this wants to be `extent` and not `extend`.

Reply:
Fixed.